### PR TITLE
Support vue.config.js file

### PR DESCRIPTION
**Pull request by Finrod927**
Vue CLI 3 added a `vue.config.js` file to config Vue projects.
This PR add the Vue icon for this file.

### CODE REVIEW

File: config.cson [hunk 1]
```diff
@@ -5240,8 +5240,10 @@ fileIcons:
 	Vue:
 		icon: "vue"
 		scope: "vue"
-		match: ".vue"
-		colour: "light-green"
+		match: [
+			["vue.config.js", "light-green"]
```
Comment (Alhadis) [inline config.cson]: This line needs to be

~~~cson
[/^vue\.config\.js$/i, "light-green", priority: 2]
~~~

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 [Unpublished]
 ----------------------
 - **New icons:** Asymptote (`.asy`), San (`.san`)
-- **Support:** Mocha (`.mocharc.{js,json,yml}`), TeX (`.pgf`, `.tikz`), Xcode (`.xccheckout`, `.xcscheme`, `.xcscmblueprint`, `.xcsettings`, `.xcuserstate`, `.xcworkspacedata`)
+- **Support:** Mocha (`.mocharc.{js,json,yml}`), TeX (`.pgf`, `.tikz`), Xcode (`.xccheckout`, `.xcscheme`, `.xcscmblueprint`, `.xcsettings`, `.xcuserstate`, `.xcworkspacedata`), Vue (`vue.config.js`, `.vue`)
```
Comment (Alhadis) [inline CHANGELOG.md]: Please remove `.vue` from this list.

**Comment by Finrod927** [comment]
Done :)

**Comment by Alhadis** [comment]
Thanks!

**Pull request merged**